tox.ini

```diff
@@ -11,4 +11,4 @@ deps =
     -rtest-requirements.txt
 
 commands =
-    pylint: pylint graph_playbook.py
+    pylint: pylint src/ansible_repo_grapher/graph_playbook.py
```

Comment:
I did try adding it. The reason I didn't leave it in is pylint found a number of similar things in both main files and would "fail" on that.